- fixed startup error on windows

ksv87 opened the pull request:
- added verification of the existence of the configuration file, if the file does not exist, DNS servers 8.8.8.8 and 8.8.4.4 are used, no additional configuration files are generated
- https://github.com/zmap/zdns/issues/125

ksv87 committed:
- fixed startup error on windows
- added verification of the existence of the configuration file, if the file does not exist, DNS servers 8.8.8.8 and 8.8.4.4 are used, no additional configuration files are generated

zakird commented on lookup.go:
```diff
@@ -34,17 +35,21 @@ type routineMetadata struct {
-	for _, s := range c.Servers {
-		if s[0:1] != "[" && strings.Contains(s, ":") {
-			s = "[" + s + "]"
+	if _, err := os.Stat(path); errors.Is(err, os.ErrNotExist) {
```
I don't think that it makes sense for defaults to be specified deep in a function like this. Rather, they should be specified as default values elsewhere, e.g., https://github.com/zmap/zdns/blob/master/zdns/main.go. It also likely makes more sense for them to be added in if the call to `GetDNSServers` fails, rather than being specified in the call itself. Can you investigate trying to move this check upwards in the ZDNS stack?

ksv87 replied:
Ok

ksv87 replied:
done

ksv87 replied:
@zakird Did I do wrong?

ksv87 committed:
- fixed startup error on windows
- added verification of the existence of the configuration file, if the file does not exist, DNS servers 8.8.8.8 and 8.8.4.4 are used, no additional configuration files are generated

zakird committed:
Adding warning if unable to find default system resolvers

zakird reviewed (approved):
Looks good. Added a quick warning and declared as const.

ksv87 commented:
![image](https://user-images.githubusercontent.com/15774585/144082929-9de6f879-b6fa-469e-9511-e6df99aaeb2b.png)
travis-ci.org it no longer works, and you need a check

zakird committed:
Merge branch 'master' into master
